### ISSUE TITLE
Regions in Thorium seem puzzling

### ISSUE THREAD

**Issue by spudthebud** [edrlab/thorium-reader#1598]
I've was stepping through Thorium 1.7.3 with JAWS 2021 and noticed some things with the regions.

1) a region called toolbar contains all the other regions, including the main region.
2) there are two toolbar regions (parent and child), so it's confusing. What's the difference between them?
3) there are two navigation regions labelled bookshelf menu. What's the difference between them? Why two?
4) there is a main region labelled "main content", which is redundant. I imagine leaving it an unlabeled "main" would be sufficient for users.
5) The main content Main region contains a region called "main content" . It's unclear to me why that's needed. Shouldn't the main region be sufficient?

Overall, my impression is that this organization could be a source of confusion. 

The image that follows depicts the region list dialog box in JAWS, when activated with JAWS + Control + R

![image](https://user-images.githubusercontent.com/8799391/144711057-7a826a29-1583-467a-adfb-97076b7129ba.png)

**Comment by danielweck** [comment]
Thank you @spudthebud I am going to merge the other "region" issue here, to avoid duplication.

**Comment by danielweck** [comment]
Merging issue https://github.com/edrlab/thorium-reader/issues/1601

---

I'm studying Thorium with the JAWS screen reader, although I imagine it's true for NVDA too. 

I'm looking at the use of regions to get my orientation.

I notice that in Reading View and Bookshelf View, there is a "main content" region within a "main content" main region. The only bit of content in the "main content" region is a line of text that says "main content".  I find this quite confusing.  

Here's a sketch:

Bookshelf view

"Main Content" main region (start)
.....  "main content" region (start)
..... ..... main content (static text)
..... "main content" region  (end)
..... Heading 2, Resume Reading
etc.
"Main Content" main region (end)

**Comment by danielweck** [comment]
Hello, I tweaked the reader window's roles and accessible labels and I tested with VoiceOver on the latest version of MacOS. I didn't test with other configurations, could you please test the latest alpha which is currently building, see the status link to track progress (the Windows installer should be ready within the next half hour): https://github.com/edrlab/thorium-reader/actions/runs/1584412998

**Comment by spudthebud** [comment]
I'm not sure the steps in getting the Alpha version

**Comment by spudthebud** [comment]
@danielweck just following up about where I can download the alpha with the changes to regions.

**Comment by spudthebud** [comment]
@danielweck thank you. I've tried it. It feels a lot better.  Now that I'm spending less time puzzling over the regions, I think I'm getting a better grasp of the other Thorium features.

**Comment by danielweck** [comment]
Thank you for your feedback, please do not hesitate to post your analysis again if needed.
All the keyboard shortcuts are described in this wiki page: https://github.com/edrlab/thorium-reader/wiki/Keyboard-shortcuts
I think you will find SHIFT CTRL N useful (to jump into the table of contents, directly at the heading near the current reading location).
There is also SHIFT CTRL I to open the "where am I" information, including a trail of headings that lead to the current reading location.
The main challenge is to make sure that your screen reader sets the correct current reading location. By default, screen readers use their own reading cursor, independently from what the web browser engine registers. In our tests with NVDA Jaws and VoiceOver, we are able to trigger a mouse click on the current screen reader cursor, which emulates a sighted user clicking on the document text to inform the current reading location. I think that KEY SPACE or KEY ENTER is the keyboard shortcut that triggers the mouse click.
Let us know how it works for you. Regards, Daniel
